shared/tx-pool-executor/src/tx_pool_executor.rs

```diff
@@ -109,8 +115,9 @@ impl<CS: ChainStore> TxPoolExecutor<CS> {
                 let verified_result = TransactionVerifier::new(
                     &tx,
                     &block_median_time_context,
-                    tip_number,
+                    parent_number + 1,
```

Comment:
This is a bug fix?

Reply:
Yes, it is a bugfix https://github.com/nervosnetwork/ckb/pull/1150

I should rebase #1150 after it merged.